consensus/reactor_test.go

```diff
@@ -65,6 +65,12 @@ func TestValidatorSetChanges(t *testing.T) {
 	nPeers := 8
 	nVals := 4
 	css := randConsensusNetWithPeers(nVals, nPeers, "consensus_val_set_changes_test", newMockTickerFunc(true))
+
+	// otherwise, the test is failing with timeout error
```

Comment:
@ebuchman do you want me to write another test with 4 validators and `skip_timeout_commit` set to false?